worker-store-fs/src/main/java/com/hpe/caf/worker/datastore/fs/FileSystemDataStore.java

```diff
@@ -45,6 +54,9 @@
     private final AtomicInteger numDx = new AtomicInteger(0);
     private final DataStoreMetricsReporter metrics = new FileSystemDataStoreMetricsReporter();
     private static final Logger LOG = LoggerFactory.getLogger(FileSystemDataStore.class);
+    private final Callable<HealthResult> healthcheck;
+    private final Duration healthcheckTimeout;
+    private final ExecutorService healthcheckExecutor = Executors.newSingleThreadExecutor();
```

Comment:
It's OK since there are already other variables being initialized in this way but just for the record the initialization should be being done in the constructor.